service/quote-server/src/tee.rs

```diff
@@ -63,10 +63,6 @@ fn generate_tdx_report_data(
 }
 
 fn get_tdx_quote(report_data: Option<String>, nonce: String) -> Result<String> {
-    if nonce.is_empty() {
```

Comment:
So the nonce can be empty?

Reply:
yes, put all the possibility there, and the hash logic to generate tdx report data is also enhanced to allow empty nonce

Reply:
Got it.